### PR TITLE
Adapter code for pune- flood data

### PR DESCRIPTION
**Pull request by DivyaSreeMunagavalasa**

### CODE REVIEW

**Comment by DivyaSreeMunagavalasa** [comment]
Handled the conflicts of <= and >= in attribute query, hence resolved the multi value attribute query issue.

**Review by ankitmashu — APPROVED** [review]
LGTM

File: examples/consumers(adapter)/rpc_adapter_pune_live.py [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+        return Q('range', **{field: {operator: value}})
+
+def build_attribute_query(attribute_query_params):
+    if not attribute_query_params:
+        logging.info("No query params..")
+        return None
+
+    attr_query = attribute_query_params
+
+    if ';' in attr_query:
```
[the remark this answers is not in the document]
Comment (DivyaSreeMunagavalasa) [reply examples/consumers(adapter)/rpc_adapter_pune_live.py:359]: Handled the conflicts of <= and >= in attribute query, hence resolved the multi value attribute query issue.

**Review by pranavrd — APPROVED** [review]
LGTM

**Pull request merged**